Makefile

```diff
@@ -222,7 +222,6 @@ config-libxml2:
 	@echo
 	$(MKDIR) 3rdParty/libxml2/$(INSTALL_DIR)
 	cd 3rdParty/libxml2 && ./autogen.sh --prefix="$(ROOT_DIR)/3rdParty/libxml2/$(INSTALL_DIR)" --without-python $(HOST_CROSS_TRIPLE) && $(MAKE) && $(MAKE) install
-	cd 3rdParty/libxml2 && git clean -fdx -e 'install/' || true
```

Comment:
@adrpo If this is not cleaned up, then it must use different directories for each target (mingw, mac, linux, ...).

Reply:
Cleaning it was actually just a workaround ...